new-dti-website/components/products/products.json

```diff
@@ -18,20 +18,8 @@
           "intensity": 0.4
         }
       ],
-      "images": [
-        {
-          "src": "/images/products/cureviews/CuReviews1.png",
-          "alt": "CU Reviews Image 1"
-        },
-        {
-          "src": "/images/products/cureviews/CuReviews2.png",
-          "alt": "CU Reviews Image 2",
-          "bottom": "-5%",
-          "right": "15%",
-          "width": "60%",
-          "direction": "down"
-        }
-      ]
+      "productImage": "/images/products/static/cureviews.png",
+      "productImageDimensions": 1500
     },
     {
       "alt": "courseplan",
```

Comment:
can we make the alt text a bit descriptive?

For example this is what Apple does for similar product mockups:
<img width="1525" alt="Screenshot 2024-12-05 at 10 03 58 AM" src="https://github.com/user-attachments/assets/bb99248d-3476-467a-b974-2f50dc67f0d5">


 I'll provide some suggestions in the review